tests/infer/test_inference.py

```diff
@@ -1005,3 +1005,72 @@ def guide(data, weights):
         loss = svi.step(data, weights)
         if step % 20 == 0:
             logger.info("step {} loss = {:0.4g}".format(step, loss))
+
+
+@pytest.mark.stage("integration", "integration_batch_2")
+class OneWayNormalRandomEffects(TestCase):
```

Comment:
how much time do these tests take?

Reply:
> how much time do these tests take?

In CI :
|                                 | Time [sec] |
|---------------------------------|------------|
| `test_renyi_nonreparameterized` | 50.67         |
| `test_renyi_reparameterized`    | 36.59         |
| `test_renyi_vectorized`         | 36.32        |

So all are in the top ten slowest for batch 2.